src/main/java/racingcar/controller/RacingController.java

```diff
@@ -0,0 +1,68 @@
+package racingcar.controller;
+
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+import racingcar.dto.CarDto;
+import racingcar.service.ParseCarNames;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+import java.util.List;
+
+public class RacingController {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final ParseCarNames parseCarNames;
+
+    public RacingController(InputView inputView, OutputView outputView, ParseCarNames parseCarNames) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.parseCarNames = parseCarNames;
+    }
+
+    public void run() {
+        Cars cars = RacingStart();
+
+        moveCar(cars);
+
+        List<String> winner = getWinner(cars);
+
+        printWinner(winner);
+
+        inputView.closeConsole();
+    }
+
+    public void printWinner(List<String> winner){
+        outputView.printWinner(winner);
+    }
+
+    public List<String> getWinner(Cars cars) {
+        List<CarDto> winner = cars.findWinner();
+        return winner.stream().
+                map(CarDto::getName)
+                .toList();
+    }
+
+    public void moveCar(Cars cars) {
+        outputView.moveNumber();
+        int tryMove = inputView.getCarMoveNumber();
+        if(tryMove<=0){
+            throw new IllegalArgumentException();
+        }
```

Comment:
이름 일관성을 맞춰보셔도 좋을 것 같아요.
그리고 `tryMove`보다는 `횟수`라는 것이 더 드러나는 이름이면 좋겠네요

```suggestion
        int moveCount = inputView.readMoveCount();
        if (moveCount <= 0) {
            throw new IllegalArgumentException("음수 혹은 0을 입력할 수 없습니다.");
        }
```